xdsl/dialects/riscv.py

```diff
@@ -1946,7 +1948,7 @@ class JalrOp(RdRsImmJumpOperation):
     """
     Jump to address and place return address in rd.
 
-    ```
+    ``` C
```

Comment:
Does this not have to be 
```
```C
```
To work correctly?

Reply:
hmm, not in the markdown editors I use but you might be right